src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -2276,6 +2286,18 @@ quantitykind:Count
   rdfs:label "Count"@en ;
   skos:broader quantitykind:Dimensionless .
 
+quantitykind:CountRate
```

Comment:
We already have the skos:broader hierarchy with InverseTime at the top, and which includes Incidence. Are you thinking we need this as well?

Reply:
I do. Incidence is apparently from epidemiology, this is more general. Could it be a broader kind of incidence?

Reply:
I can go along with the skos:broader you suggest.